hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -652,7 +652,27 @@ public void updateRpc(MethodDescriptor method, Message param, CallStats stats, T
       concurrentCallsPerServerHist.update(callsPerServer);
     }
     // Update the counter that tracks RPCs by type.
-    final String methodName = method.getService().getName() + "_" + method.getName();
+    String methodName = method.getService().getName() + "_" + method.getName();
```

Comment:
Better use a StringBuilder here and finally build the methodName through StringBuilder.toString?

Reply:
Resolved it.